web-app/js/campaigns/contestApplication.js

```diff
@@ -62,14 +62,19 @@ var contestApplicationFunctions = {
                 if (contestApplicationVote.success) {
                     console.log("Vote Success")
                     console.log(contestApplicationVote)
-                    var $buttonMainPage = $("section#main .leader-post .contestApplication-vote")
-                    var $buttonColumnC = $("#aside-ppal .call-to-action .actions a")
-                    var $liStats = $(".leader-post-stats .fa-scroll").parents("li")
+                    const $buttonMainPage = $("section#main .leader-post .contestApplication-vote")
+                    const $buttonColumnC = $("#aside-ppal .call-to-action .actions .contestApplication-vote a")
+                    const $buttonContestApplicationList = $("#contest-applications-list  .contestApplication-vote-" + contestApplicationVote.vote.contestApplicationId)
+                    const $liStats = $(".leader-post-stats .fa-scroll").parents("li")
 
-                    contestApplicationFunctions._disableButton($button, contestApplicationVote.vote.votes);
-                    contestApplicationFunctions._disableButton($buttonMainPage, contestApplicationVote.vote.votes);
-                    contestApplicationFunctions._disableButton($buttonColumnC, contestApplicationVote.vote.votes);
-                    contestApplicationFunctions._disableButton($liStats, contestApplicationVote.vote.votes);
+                    if($button.attr("data-loggedUser") !== '') {
+                        contestApplicationFunctions._disableVoteButton($buttonMainPage, contestApplicationVote.vote.votes);
+                        contestApplicationFunctions._disableVoteButton($buttonColumnC, contestApplicationVote.vote.votes);
+                        contestApplicationFunctions._disableVoteButton($buttonContestApplicationList, contestApplicationVote.vote.votes);
+                    }
+                    contestApplicationFunctions._updateNumVotes($buttonMainPage, contestApplicationVote.vote.votes);
+                    contestApplicationFunctions._updateNumVotes($liStats, contestApplicationVote.vote.votes);
+                    contestApplicationFunctions._updateNumVotes($buttonContestApplicationList, contestApplicationVote.vote.votes);
```

Comment:
For improvung the readabilty, could be in a function called "updateButtonsAfterSuccess" or something like this